RFC-010-core-strategy/RFC-010-core-strategy.md

```diff
@@ -0,0 +1,126 @@
+
+
+## Motivation
+
+There are several benefits to articulating where we are, how we can best support our users and community members, and where we’d like to grow in future.
+
+Documenting our core strategy means that:
+
+**As a project**, we can: 
+
+* Have a touchstone or “North Star” to guide decisions about content produced by the project, as well as communications about the project.
+* Reinforce the foundation that is already underlying our current initiatives.
+* Adapt easily to support our audience. As we grow, our audience will grow and their needs might change. 
+
+**As a community**, we can:
+
+* Hold a shared idea of what we’re about—so that we’re on the same page. 
+* Present a unified idea, and use consistent language, when talking about the identity of our project to others.
+
+**As content producers**, we can: 
+
+* Address content challenges, prioritize them accordingly and form solutions. 
+* Tie each piece of content that we create back to a community or project goal. 
+
+
+## Proposal
+
+That we accept the following related file:
+
+* core-strategy.md
+
+
+## Consequences
+
+An agreed core strategy will help all members of our e annually, or if we happen to have a major shift in focus or experience unexpected growth.
+
+
+## Links and prior art
+
+{This section is optional if you want to [link](https://example.com) to other resources.}
+
+
+## Open questions
+
+{This section is optional and is where you can list questions that won't be resolved by this RFC, including those raised in comments from community members.}
+
+
+## Decisions deferred
+
+What’s not covered is the deeper dive into the various pillars of content strategy such as content substance, workflow, structure and governance. 
+These pillars answer questions like:
+
+* What are our content types?
+* Who produces and maintains our content?
+* On what channels and platforms does the content live?
+* How do our users access the content?
+* When and how is content reviewed, evaluated and archived?
+
+This work is intended to follow as future deliverables from the Content Strategy working group.
+
+
+## Feedback
+
+{If you accept feedback from a community member, you will incorporate it into your RFC before it is accepted.
+If you reject feedback, note that rejected feedback here before resolving the conversation.}
+
+
+## Implementation checklist
+
+If this proposal is accepted, the following tasks must be completed:
+
+- [ ] Task number one
+- [ ] Task number two
+- [ ] Task number three, etc.
+
+
+## Votes
+
+Votes as per our [decision process](https://thegooddocsproject.dev/decisions/):
+
+Project steering committee (listed alphabetically by first name):
+
+- Aaron Peters:
+- Aidan Doherty:
+- Alyssa Rock:
+- Ankita Tripathi:
+- Bryan Klein:
+- Cameron Shorter:
+- Carrie Crowe:
+- Erin McKean:
+- Deanna Thompson:

```

Comment:
I am the newest member of the PSC, so I would be glad if you added my name as well (Tina Lüdtke).
<details><summary>Side note</summary>
Not to be too nitpicky but the list isn't ordered alphabetically. 😬 
</details>

Reply:
Will add here and fix in the template.